### DISCUSSION TITLE
DataCollection validation

### DISCUSSION THREAD

**Discussion by RazorMeister** [spatie/laravel-data#185] [Q&A]
Hello, recently I've encountered strange situation.

I have UdpateWarehouseDto with DataCollection property:

```
class UpdateWarehouseDto extends BaseWarehouseDto
{
    #[DataCollectionOf(UpdateWarehouseOpeningDto::class)]
    #[Size(7)]
    public DataCollection $openings;
}
````

Example request json:
```
{
    "openings": [
        { ... },
        { ... },
        { ... }
     ]
}
```

Unfortunately this `Size` attribute is completely ignored. In fact all rules applied to this field are ignored. 
Is this intentional?

**Comment by rubenvanassche** [comment]
Good point, fixed this in 2.0.5!
